tests/test_middleware_interface.py

```diff
@@ -286,6 +291,7 @@ def test_run_pipeline(self):
             self.interface.run_pipeline(self.next_visit, {1})
         mock_run.assert_called_once_with(register_dataset_types=True)
 
+    @unittest.skip("Snap ID argument currently unused.")
```

Comment:
rephrase: "run_pipeline ignores the exposure_ids arg, so we can't test passing in invalid ones."